bundles/tools.vitruv.change.propagation/src/tools/vitruv/change/propagation/ChangeRecordingModelRepository.java

```diff
@@ -28,8 +29,8 @@ public interface ChangeRecordingModelRepository extends ResourceAccess, AutoClos
 	 * 
 	 * @param changeApplicator - the function applying changes to the models, must
 	 *                         not be <code>null</code>
-	 * @return the list of {@link ChangeInPropagation} containing the performed
-	 *         changes , one for each metamodel
+	 * @return the list of {@link TransitiveChange} containing the performed
+	 *         changes, one for each metamodel
 	 */
-	Iterable<ChangeInPropagation> recordChanges(Runnable changeApplicator);
+	Iterable<TransactionalChange> recordChanges(Runnable changeApplicator);
```

Comment:
`TransitiveChange` or `TransactionalChange`? ;)

Reply:
`TransactionalChange` is actually correct :-) There is no `TransitiveChange`. A `TransactionalChange` currently has the semantics that is must be executed as a whole (for whatever reason, correctness of the resulting model or whatever).

Reply:
Yep, but the documentation says `TransitiveChange`

Reply:
Fixed in d7fba5e